panic: runtime error: invalid memory address or nil pointer dereference

This is my first time trying to spin up VMs with terraform, so please do forgive any ignorance.  Our VM templates are in the "Templates" folder, I don't know if this may be part of the issue.  Below is the config I am using, along with the error output.

```
provider "vsphere" {
  user = "secret_username"
  password = "secret_password"
  vcenter_server = "secret_host"
}

resource "vsphere_virtual_machine" "default" {
    name = "secret_hostname"
    datacenter = "secret_datacenter"
    cluster = "secret_cluster"
    datastore = "secret_emc"
    template = "linux-vm-template"
    vcpu = 2
    memory = 2048
    gateway = "10.1.1.1"
    network_interface {
        device_name = "eth1"
        label = "Private-VLAN-1"
        ip_address = "10.1.1.10"
        subnet_mask = "255.255.255.0"
    }
}
```

And here is the debug output, redacted of sensitive information
```
2015/03/17 14:47:47 [INFO] Terraform version: 0.3.7  0bc0c03fece07c4f21ee5195743116a4d418f234+CHANGES
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: atlas = secret_home_path/Downloads/terraform/terraform-provider-atlas
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: aws = secret_home_path/Downloads/terraform/terraform-provider-aws
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: cloudflare = secret_home_path/Downloads/terraform/terraform-provider-cloudflare
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: cloudstack = secret_home_path/Downloads/terraform/terraform-provider-cloudstack
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: consul = secret_home_path/Downloads/terraform/terraform-provider-consul
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: digitalocean = secret_home_path/Downloads/terraform/terraform-provider-digitalocean
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: dnsimple = secret_home_path/Downloads/terraform/terraform-provider-dnsimple
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: google = secret_home_path/Downloads/terraform/terraform-provider-google
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: heroku = secret_home_path/Downloads/terraform/terraform-provider-heroku
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: mailgun = secret_home_path/Downloads/terraform/terraform-provider-mailgun
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: null = secret_home_path/Downloads/terraform/terraform-provider-null
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: vsphere = secret_home_path/Downloads/terraform/terraform-provider-vsphere
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: file = secret_home_path/Downloads/terraform/terraform-provisioner-file
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: local-exec = secret_home_path/Downloads/terraform/terraform-provisioner-local-exec
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: remote-exec = secret_home_path/Downloads/terraform/terraform-provisioner-remote-exec
2015/03/17 14:47:47 Detected home directory from env var: secret_home_path
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: atlas = secret_home_path/Downloads/terraform/terraform-provider-atlas
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: aws = secret_home_path/Downloads/terraform/terraform-provider-aws
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: cloudflare = secret_home_path/Downloads/terraform/terraform-provider-cloudflare
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: cloudstack = secret_home_path/Downloads/terraform/terraform-provider-cloudstack
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: consul = secret_home_path/Downloads/terraform/terraform-provider-consul
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: digitalocean = secret_home_path/Downloads/terraform/terraform-provider-digitalocean
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: dnsimple = secret_home_path/Downloads/terraform/terraform-provider-dnsimple
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: google = secret_home_path/Downloads/terraform/terraform-provider-google
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: heroku = secret_home_path/Downloads/terraform/terraform-provider-heroku
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: mailgun = secret_home_path/Downloads/terraform/terraform-provider-mailgun
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: null = secret_home_path/Downloads/terraform/terraform-provider-null
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: vsphere = secret_home_path/Downloads/terraform/terraform-provider-vsphere
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: file = secret_home_path/Downloads/terraform/terraform-provisioner-file
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: local-exec = secret_home_path/Downloads/terraform/terraform-provisioner-local-exec
2015/03/17 14:47:47 [DEBUG] Discoverd plugin: remote-exec = secret_home_path/Downloads/terraform/terraform-provisioner-remote-exec
2015/03/17 14:47:47 Detected home directory from env var: secret_home_path
2015/03/17 14:47:47 [DEBUG] Attempting to open CLI config file: secret_home_path/.terraformrc
2015/03/17 14:47:47 [DEBUG] File doesn't exist, but doesn't need to. Ignoring.
2015/03/17 14:47:47 Detected home directory from env var: secret_home_path
2015/03/17 14:47:47 [DEBUG] Creating graph for path: [secret_user]
2015/03/17 14:47:47 [DEBUG] Starting plugin: secret_home_path/Downloads/terraform/terraform-provider-vsphere []string{"secret_home_path/Downloads/terraform/terraform-provider-vsphere"}
2015/03/17 14:47:47 [DEBUG] Waiting for RPC address for: secret_home_path/Downloads/terraform/terraform-provider-vsphere
2015/03/17 14:47:47 terraform-provider-vsphere: 2015/03/17 14:47:47 Plugin address: unix /var/folders/s6/3j2rdcmn4tg57f2d9x54l_kc0000gp/T/tf-plugin385387694
2015/03/17 14:47:47 [DEBUG] Graph [secret_user] created and valid. 3 nouns.
2015/03/17 14:47:47 [DEBUG] Creating graph for path: [secret_user]
2015/03/17 14:47:47 [DEBUG] Graph [secret_user] created and valid. 3 nouns.
2015/03/17 14:47:47 [INFO] Validating provider: vsphere
2015/03/17 14:47:47 [DEBUG] vsphere_virtual_machine.default: expanding to count = 1
2015/03/17 14:47:47 [INFO] Validating resource: vsphere_virtual_machine.default
2015/03/17 14:47:47 [DEBUG] Creating graph for path: [secret_user]
2015/03/17 14:47:47 [DEBUG] Graph [secret_user] created and valid. 3 nouns.
2015/03/17 14:47:47 [INFO] Configuring provider: vsphere
2015/03/17 14:47:47 terraform-provider-vsphere: 2015/03/17 14:47:47 [INFO] VMWare vSphere Client configured for URL: https://secret_user:secret_password@secret_vsphere_host/sdk
2015/03/17 14:47:47 [DEBUG] vsphere_virtual_machine.default: expanding to count = 1
2015/03/17 14:47:47 [INFO] Module secret_user walking: vsphere_virtual_machine.default (Graph node: vsphere_virtual_machine.default)
2015/03/17 14:47:47 [DEBUG] vsphere_virtual_machine.default: Not refreshing, ID is empty
2015/03/17 14:47:47 [DEBUG] Creating graph for path: [secret_user]
2015/03/17 14:47:47 [DEBUG] Graph [secret_user] created and valid. 3 nouns.
2015/03/17 14:47:47 [INFO] Configuring provider: vsphere
2015/03/17 14:47:48 terraform-provider-vsphere: 2015/03/17 14:47:48 [INFO] VMWare vSphere Client configured for URL: https://secret_user:secret_password@secret_vsphere_host/sdk
2015/03/17 14:47:48 [DEBUG] vsphere_virtual_machine.default: expanding to count = 1
2015/03/17 14:47:48 [INFO] Module secret_user walking: vsphere_virtual_machine.default (Graph node: vsphere_virtual_machine.default)
2015/03/17 14:47:48 [DEBUG] vsphere_virtual_machine.default: Executing diff
2015/03/17 14:47:48 [DEBUG] vsphere_virtual_machine.default: Diff: &terraform.InstanceDiff{Attributes:map[string]*terraform.ResourceAttrDiff{"cluster":*terraform.ResourceAttrDiff{Old:"", New:"Cluster_01", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "network_interface.0.label":*terraform.ResourceAttrDiff{Old:"", New:"Secret-VLAN-1", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "vcpu":*terraform.ResourceAttrDiff{Old:"", New:"2", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "template":*terraform.ResourceAttrDiff{Old:"", New:"centos66-template-07", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "network_interface.0.subnet_mask":*terraform.ResourceAttrDiff{Old:"", New:"255.255.255.0", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "memory":*terraform.ResourceAttrDiff{Old:"", New:"2048", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "id":*terraform.ResourceAttrDiff{Old:"", New:"", NewComputed:true, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:true, Type:0x2}, "network_interface.0.ip_address":*terraform.ResourceAttrDiff{Old:"", New:"10.1.1.10", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "name":*terraform.ResourceAttrDiff{Old:"", New:"vm-guest-name", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "network_interface.#":*terraform.ResourceAttrDiff{Old:"0", New:"1", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "datacenter":*terraform.ResourceAttrDiff{Old:"", New:"datacenter_location", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "datastore":*terraform.ResourceAttrDiff{Old:"", New:"EMC_VMAX_3TB_1", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "gateway":*terraform.ResourceAttrDiff{Old:"", New:"10.1.1.1", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}, "network_interface.0.device_name":*terraform.ResourceAttrDiff{Old:"", New:"eth1", NewComputed:false, NewRemoved:false, NewExtra:interface {}(nil), RequiresNew:false, Type:0x0}}, Destroy:false, DestroyTainted:false}
2015/03/17 14:47:48 [INFO] Apply walk starting
2015/03/17 14:47:48 [DEBUG] Creating graph for path: [secret_user]
2015/03/17 14:47:48 [DEBUG] Graph [secret_user] created and valid. 3 nouns.
2015/03/17 14:47:48 [INFO] Configuring provider: vsphere
2015/03/17 14:47:48 terraform-provider-vsphere: 2015/03/17 14:47:48 [INFO] VMWare vSphere Client configured for URL: https://secret_user:secret_password@secret_vsphere_host/sdk
2015/03/17 14:47:48 [DEBUG] vsphere_virtual_machine.default: expanding to count = 1
2015/03/17 14:47:48 [INFO] Module secret_user walking: vsphere_virtual_machine.default (Graph node: vsphere_virtual_machine.default)
2015/03/17 14:47:48 [DEBUG] vsphere_virtual_machine.default: Executing Apply
2015/03/17 14:47:48 terraform-provider-vsphere: 2015/03/17 14:47:48 [DEBUG] network_interface init: [{eth1 Secret-VLAN-1 10.1.1.10 255.255.255.0}]
2015/03/17 14:47:49 terraform-provider-vsphere: panic: runtime error: invalid memory address or nil pointer dereference
2015/03/17 14:47:49 terraform-provider-vsphere: [signal 0xb code=0x1 addr=0x20 pc=0x4a836]
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 61 [running]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/rakutentech/terraform-provider-vsphere/vsphere.getVirtualMachine(0xc20810d080, 0xc208215050, 0xc20820a6e0, 0x14, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/rakutentech/terraform-provider-vsphere/vsphere/virtual_machine.go:301 +0x116
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/rakutentech/terraform-provider-vsphere/vsphere.(*VirtualMachine).RunVirtualMachine(0xc20811eb60, 0xc20810d080, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/rakutentech/terraform-provider-vsphere/vsphere/virtual_machine.go:69 +0x38e
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/rakutentech/terraform-provider-vsphere/vsphere.resourceVSphereVirtualMachineCreate(0xc2081a5380, 0xaa87a0, 0xc20810d080, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/rakutentech/terraform-provider-vsphere/vsphere/resource_vsphere_virtual_machine.go:179 +0x1595
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/helper/schema.(*Resource).Apply(0xc208143b40, 0xc2082144e0, 0xc208210c00, 0xaa87a0, 0xc20810d080, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/helper/schema/resource.go:145 +0x2c1
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/helper/schema.(*Provider).Apply(0xc208178240, 0xc2081f30c0, 0xc2082144e0, 0xc208210c00, 0x5ba060, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/helper/schema/provider.go:154 +0x1da
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.(*ResourceProviderServer).Apply(0xc208189120, 0xc2081f1e40, 0xc208210e70, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/resource_provider.go:319 +0x7f
2015/03/17 14:47:49 terraform-provider-vsphere: reflect.Value.call(0x7f6380, 0xa84860, 0x13, 0xacf770, 0x4, 0xc20823ff28, 0x3, 0x3, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/reflect/value.go:419 +0x10e5
2015/03/17 14:47:49 terraform-provider-vsphere: reflect.Value.Call(0x7f6380, 0xa84860, 0x13, 0xc20823ff28, 0x3, 0x3, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/reflect/value.go:296 +0xbc
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*service).call(0xc208143bc0, 0xc208143b80, 0xc208073508, 0xc208103c80, 0xc2081896e0, 0x5ba000, 0xc2081f1e40, 0x16, 0x5ba060, 0xc208210e70, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:382 +0x1f7
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/rpc.(*Server).ServeCodec
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:476 +0x44a
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 1 [IO wait]:
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).Wait(0xc208010140, 0x72, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).WaitRead(0xc208010140, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*netFD).accept(0xc2080100e0, 0x0, 0x145bd20, 0xc20800ae58)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_unix.go:419 +0x40b
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*UnixListener).AcceptUnix(0xc20801ec20, 0x38360, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/unixsock_posix.go:282 +0x56
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*UnixListener).Accept(0xc20801ec20, 0x0, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/unixsock_posix.go:293 +0x4c
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.(*Server).Accept(0xc20800ab80, 0x145c598, 0xc20801ec20)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:33 +0x37
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/plugin.Serve(0xc20800ab70)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/plugin/server.go:88 +0x7e8
2015/03/17 14:47:49 terraform-provider-vsphere: main.main()
2015/03/17 14:47:49 terraform-provider-vsphere: 	secret_home_path/Downloads/terraform/terraform-provider-vsphere/main.go:11 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 5 [syscall]:
2015/03/17 14:47:49 terraform-provider-vsphere: os/signal.loop()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
2015/03/17 14:47:49 terraform-provider-vsphere: created by os/signal.init·1
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 7 [chan receive]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/plugin.func·007()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/plugin/server.go:79 +0x69
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/plugin.Serve
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/plugin/server.go:85 +0x7c4
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 8 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc208066280, 0xc20802b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc20805e5a0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc20805e5a0, 0xc20800af50, 0x1, 0x9, 0x2592, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc20805e5a0, 0xc20800af50, 0x1, 0x9, 0x1, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc20805e5a0, 0xc20800af50, 0x1, 0x9, 0x2c, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.decodeUintReader(0x145d6f0, 0xc20805e5a0, 0xc20800af50, 0x9, 0x9, 0x0, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decode.go:121 +0x99
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).recvMessage(0xc208102580, 0xc208118a20)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:76 +0x55
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).decodeTypeSequence(0xc208102580, 0xc20801f400, 0x16)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:140 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).DecodeValue(0xc208102580, 0x5f4f20, 0xc20801f400, 0x16, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:208 +0x192
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).Decode(0xc208102580, 0x5f4f20, 0xc20801f400, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:185 +0x297
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*gobServerCodec).ReadRequestHeader(0xc20810a240, 0xc20801f400, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:402 +0x5e
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequestHeader(0xc208053880, 0x145d850, 0xc20810a240, 0x0, 0x0, 0xc20801f400, 0xc20800af00, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:575 +0x97
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequest(0xc208053880, 0x145d850, 0xc20810a240, 0xc208102480, 0xc20801ed80, 0x616e20, 0x0, 0x0, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:542 +0xa2
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeCodec(0xc208053880, 0x145d850, 0xc20810a240)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:461 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeConn(0xc208053880, 0x145d778, 0xc208066280)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:453 +0x208
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.(*Server).ServeConn(0xc20800ab80, 0x145d690, 0xc208030078)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:76 +0x530
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*Server).Accept
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:39 +0x173
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 9 [IO wait]:
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).Wait(0xc208010220, 0x72, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).WaitRead(0xc208010220, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*netFD).Read(0xc2080101c0, 0xc20800f000, 0x1000, 0x1000, 0x0, 0x145bd20, 0xc2082108f0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_unix.go:242 +0x40f
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*conn).Read(0xc208030078, 0xc20800f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/net.go:121 +0xdc
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc20805e240)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc20805e240, 0xc20800ae70, 0xc, 0xc, 0xc2082108e0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc20805e240, 0xc20800ae70, 0xc, 0xc, 0xc, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc20805e240, 0xc20800ae70, 0xc, 0xc, 0xc, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Session).recv(0xc208064c80)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:340 +0xcb
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/yamux.newSession
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:93 +0x3f0
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 10 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Session).send(0xc208064c80)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:298 +0x554
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/yamux.newSession
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:94 +0x40a
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 11 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Session).keepalive(0xc208064c80)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:254 +0x106
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/yamux.newSession
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:96 +0x435
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 13 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Session).AcceptStream(0xc208064c80, 0xc200000005, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/session.go:162 +0x173
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.(*muxBroker).Run(0xc20801ed40)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/mux_broker.go:107 +0x37
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*Server).ServeConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:65 +0x388
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 15 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc208066500, 0xc208133000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc20805efc0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc20805efc0, 0xc20800b8a0, 0x1, 0x9, 0xc208165858, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc20805efc0, 0xc20800b8a0, 0x1, 0x9, 0x1, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc20805efc0, 0xc20800b8a0, 0x1, 0x9, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.decodeUintReader(0x145d6f0, 0xc20805efc0, 0xc20800b8a0, 0x9, 0x9, 0x0, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decode.go:121 +0x99
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).recvMessage(0xc208102a80, 0xc208119680)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:76 +0x55
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).decodeTypeSequence(0xc208102a80, 0xc20801f700, 0x16)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:140 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).DecodeValue(0xc208102a80, 0x5f4f20, 0xc20801f720, 0x16, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:208 +0x192
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).Decode(0xc208102a80, 0x5f4f20, 0xc20801f720, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:185 +0x297
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*gobServerCodec).ReadRequestHeader(0xc20810b7a0, 0xc20801f720, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:402 +0x5e
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequestHeader(0xc20812e2c0, 0x145d850, 0xc20810b7a0, 0x0, 0x0, 0xc20801f720, 0xc20800b800, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:575 +0x97
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequest(0xc20812e2c0, 0x145d850, 0xc20810b7a0, 0xc208102800, 0xc20801fcc0, 0x5ba1e0, 0x0, 0x0, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:542 +0xa2
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeCodec(0xc20812e2c0, 0x145d850, 0xc20810b7a0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:461 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeConn(0xc20812e2c0, 0x145d778, 0xc208066500)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:453 +0x208
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.serve(0x145d778, 0xc208066500, 0xb6c030, 0x10, 0xa847c0, 0xc20801f700)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:146 +0x1f9
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.func·001()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:102 +0x265
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*dispenseServer).ResourceProvider
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:103 +0x107
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 21 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc208066780, 0xc20807d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc20805fc20)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc20805fc20, 0xc208072520, 0x1, 0x9, 0xc208167858, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc20805fc20, 0xc208072520, 0x1, 0x9, 0x1, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc20805fc20, 0xc208072520, 0x1, 0x9, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.decodeUintReader(0x145d6f0, 0xc20805fc20, 0xc208072520, 0x9, 0x9, 0x0, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decode.go:121 +0x99
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).recvMessage(0xc208103080, 0xc2081198c0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:76 +0x55
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).decodeTypeSequence(0xc208103080, 0xc20806ec00, 0x16)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:140 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).DecodeValue(0xc208103080, 0x5f4f20, 0xc20806ec40, 0x16, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:208 +0x192
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).Decode(0xc208103080, 0x5f4f20, 0xc20806ec40, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:185 +0x297
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*gobServerCodec).ReadRequestHeader(0xc208135470, 0xc20806ec40, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:402 +0x5e
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequestHeader(0xc20812f180, 0x145d850, 0xc208135470, 0x0, 0x0, 0xc20806ec40, 0xc208072500, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:575 +0x97
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequest(0xc20812f180, 0x145d850, 0xc208135470, 0xc208103000, 0xc20806e6a0, 0x5ba3c0, 0x0, 0x0, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:542 +0xa2
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeCodec(0xc20812f180, 0x145d850, 0xc208135470)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:461 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeConn(0xc20812f180, 0x145d778, 0xc208066780)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:453 +0x208
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.serve(0x145d778, 0xc208066780, 0xb6c030, 0x10, 0xa847c0, 0xc20806e680)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:146 +0x1f9
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.func·001()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:102 +0x265
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*dispenseServer).ResourceProvider
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:103 +0x107
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 27 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc2080668c0, 0xc208177000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc208172360)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc208172360, 0xc208170960, 0x1, 0x9, 0x6240e0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc208172360, 0xc208170960, 0x1, 0x9, 0x1, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc208172360, 0xc208170960, 0x1, 0x9, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.decodeUintReader(0x145d6f0, 0xc208172360, 0xc208170960, 0x9, 0x9, 0x0, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decode.go:121 +0x99
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).recvMessage(0xc208103600, 0xc2080d8120)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:76 +0x55
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).decodeTypeSequence(0xc208103600, 0xc20816e800, 0x16)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:140 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).DecodeValue(0xc208103600, 0x5f4f20, 0xc20816e8a0, 0x16, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:208 +0x192
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).Decode(0xc208103600, 0x5f4f20, 0xc20816e8a0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:185 +0x297
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*gobServerCodec).ReadRequestHeader(0xc208083260, 0xc20816e8a0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:402 +0x5e
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequestHeader(0xc20808d300, 0x145d850, 0xc208083260, 0x0, 0x0, 0xc20816e8a0, 0xc208170900, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:575 +0x97
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequest(0xc20808d300, 0x145d850, 0xc208083260, 0xc208103280, 0xc20816ee40, 0xa63fe0, 0x0, 0x0, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:542 +0xa2
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeCodec(0xc20808d300, 0x145d850, 0xc208083260)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:461 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeConn(0xc20808d300, 0x145d778, 0xc2080668c0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:453 +0x208
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.serve(0x145d778, 0xc2080668c0, 0xb6c030, 0x10, 0xa847c0, 0xc20816e880)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:146 +0x1f9
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.func·001()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:102 +0x265
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*dispenseServer).ResourceProvider
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:103 +0x107
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 38 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc208066000, 0xc20817e000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc208186300)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc208186300, 0xc2081b4150, 0x1, 0x9, 0xc208087858, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc208186300, 0xc2081b4150, 0x1, 0x9, 0x1, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc208186300, 0xc2081b4150, 0x1, 0x9, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.decodeUintReader(0x145d6f0, 0xc208186300, 0xc2081b4150, 0x9, 0x9, 0x0, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decode.go:121 +0x99
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).recvMessage(0xc208103880, 0xc2080d87e0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:76 +0x55
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).decodeTypeSequence(0xc208103880, 0xc2081b6700, 0x16)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:140 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).DecodeValue(0xc208103880, 0x5f4f20, 0xc2081b6760, 0x16, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:208 +0x192
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).Decode(0xc208103880, 0x5f4f20, 0xc2081b6760, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:185 +0x297
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*gobServerCodec).ReadRequestHeader(0xc208082480, 0xc2081b6760, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:402 +0x5e
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequestHeader(0xc2081c81c0, 0x145d850, 0xc208082480, 0x0, 0x0, 0xc2081b6760, 0xc2081b4100, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:575 +0x97
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequest(0xc2081c81c0, 0x145d850, 0xc208082480, 0xc208102200, 0xc2081b60a0, 0x5ba120, 0x0, 0x0, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:542 +0xa2
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeCodec(0xc2081c81c0, 0x145d850, 0xc208082480)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:461 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeConn(0xc2081c81c0, 0x145d778, 0xc208066000)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:453 +0x208
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.serve(0x145d778, 0xc208066000, 0xb6c030, 0x10, 0xa847c0, 0xc2081b6080)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:146 +0x1f9
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.func·001()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:102 +0x265
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*dispenseServer).ResourceProvider
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:103 +0x107
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 50 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/yamux.(*Stream).Read(0xc208067180, 0xc208139000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/github.com/hashicorp/yamux/stream.go:125 +0x3ab
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc208187da0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Read(0xc208187da0, 0xc208073570, 0x1, 0x9, 0xc2081ad858, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:174 +0x26c
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadAtLeast(0x145d6f0, 0xc208187da0, 0xc208073570, 0x1, 0x9, 0x1, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:298 +0xf1
2015/03/17 14:47:49 terraform-provider-vsphere: io.ReadFull(0x145d6f0, 0xc208187da0, 0xc208073570, 0x1, 0x9, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/io/io.go:316 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.decodeUintReader(0x145d6f0, 0xc208187da0, 0xc208073570, 0x9, 0x9, 0x0, 0x1, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decode.go:121 +0x99
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).recvMessage(0xc2081bc080, 0xc2080d8900)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:76 +0x55
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).decodeTypeSequence(0xc2081bc080, 0xc208189100, 0x16)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:140 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).DecodeValue(0xc2081bc080, 0x5f4f20, 0xc208189140, 0x16, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:208 +0x192
2015/03/17 14:47:49 terraform-provider-vsphere: encoding/gob.(*Decoder).Decode(0xc2081bc080, 0x5f4f20, 0xc208189140, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/encoding/gob/decoder.go:185 +0x297
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*gobServerCodec).ReadRequestHeader(0xc2081783f0, 0xc208189140, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:402 +0x5e
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequestHeader(0xc208143b80, 0x145d850, 0xc2081783f0, 0x0, 0x0, 0xc208189140, 0xc208073500, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:575 +0x97
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).readRequest(0xc208143b80, 0x145d850, 0xc2081783f0, 0xc208103c80, 0xc2081896e0, 0x5ba000, 0x0, 0x0, 0x0, 0x0, ...)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:542 +0xa2
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeCodec(0xc208143b80, 0x145d850, 0xc2081783f0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:461 +0x6d
2015/03/17 14:47:49 terraform-provider-vsphere: net/rpc.(*Server).ServeConn(0xc208143b80, 0x145d778, 0xc208067180)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/rpc/server.go:453 +0x208
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.serve(0x145d778, 0xc208067180, 0xb6c030, 0x10, 0xa847c0, 0xc208189120)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:146 +0x1f9
2015/03/17 14:47:49 terraform-provider-vsphere: github.com/hashicorp/terraform/rpc.func·001()
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:102 +0x265
2015/03/17 14:47:49 terraform-provider-vsphere: created by github.com/hashicorp/terraform/rpc.(*dispenseServer).ResourceProvider
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/code.google.com/p/src/github.com/hashicorp/terraform/rpc/server.go:103 +0x107
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 34 [IO wait]:
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).Wait(0xc2081846f0, 0x72, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).WaitRead(0xc2081846f0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*netFD).Read(0xc208184690, 0xc208038800, 0x800, 0x800, 0x0, 0x145bd20, 0xc2081b4c80)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_unix.go:242 +0x40f
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*conn).Read(0xc208031260, 0xc208038800, 0x800, 0x800, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/net.go:121 +0xdc
2015/03/17 14:47:49 terraform-provider-vsphere: crypto/tls.(*block).readFromUntil(0xc208178750, 0x145df28, 0xc208031260, 0x5, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/crypto/tls/conn.go:454 +0xe6
2015/03/17 14:47:49 terraform-provider-vsphere: crypto/tls.(*Conn).readRecord(0xc20810c580, 0x17, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/crypto/tls/conn.go:539 +0x2da
2015/03/17 14:47:49 terraform-provider-vsphere: crypto/tls.(*Conn).Read(0xc20810c580, 0xc208180000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/crypto/tls/conn.go:904 +0x166
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.noteEOFReader.Read(0x145e3b8, 0xc20810c580, 0xc20817a7e8, 0xc208180000, 0x1000, 0x1000, 0x7d93c0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:1270 +0x6e
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*noteEOFReader).Read(0xc208196ea0, 0xc208180000, 0x1000, 0x1000, 0xc208012000, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	<autogenerated>:125 +0xd4
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc208173aa0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Peek(0xc208173aa0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:132 +0xf0
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*persistConn).readLoop(0xc20817a790)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:842 +0xa4
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/http.(*Transport).dialConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:660 +0xc9f
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 35 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*persistConn).writeLoop(0xc20817a790)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:945 +0x41d
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/http.(*Transport).dialConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:661 +0xcbc
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 45 [IO wait]:
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).Wait(0xc208184ae0, 0x72, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:84 +0x47
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*pollDesc).WaitRead(0xc208184ae0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_poll_runtime.go:89 +0x43
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*netFD).Read(0xc208184a80, 0xc208039000, 0x800, 0x800, 0x0, 0x145bd20, 0xc208199e10)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/fd_unix.go:242 +0x40f
2015/03/17 14:47:49 terraform-provider-vsphere: net.(*conn).Read(0xc208030948, 0xc208039000, 0x800, 0x800, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/net.go:121 +0xdc
2015/03/17 14:47:49 terraform-provider-vsphere: crypto/tls.(*block).readFromUntil(0xc2081344b0, 0x145df28, 0xc208030948, 0x5, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/crypto/tls/conn.go:454 +0xe6
2015/03/17 14:47:49 terraform-provider-vsphere: crypto/tls.(*Conn).readRecord(0xc20810c000, 0x17, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/crypto/tls/conn.go:539 +0x2da
2015/03/17 14:47:49 terraform-provider-vsphere: crypto/tls.(*Conn).Read(0xc20810c000, 0xc208075000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/crypto/tls/conn.go:904 +0x166
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.noteEOFReader.Read(0x145e3b8, 0xc20810c000, 0xc208144738, 0xc208075000, 0x1000, 0x1000, 0x7d93c0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:1270 +0x6e
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*noteEOFReader).Read(0xc2081ae900, 0xc208075000, 0x1000, 0x1000, 0xc208012000, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	<autogenerated>:125 +0xd4
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).fill(0xc208187440)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:97 +0x1ce
2015/03/17 14:47:49 terraform-provider-vsphere: bufio.(*Reader).Peek(0xc208187440, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/bufio/bufio.go:132 +0xf0
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*persistConn).readLoop(0xc2081446e0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:842 +0xa4
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/http.(*Transport).dialConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:660 +0xc9f
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 46 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*persistConn).writeLoop(0xc2081446e0)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:945 +0x41d
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/http.(*Transport).dialConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:661 +0xcbc
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 57 [runnable]:
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*persistConn).readLoop(0xc20813cd10)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:928 +0x9ce
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/http.(*Transport).dialConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:660 +0xc9f
2015/03/17 14:47:49 terraform-provider-vsphere:
2015/03/17 14:47:49 terraform-provider-vsphere: goroutine 58 [select]:
2015/03/17 14:47:49 terraform-provider-vsphere: net/http.(*persistConn).writeLoop(0xc20813cd10)
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:945 +0x41d
2015/03/17 14:47:49 terraform-provider-vsphere: created by net/http.(*Transport).dialConn
2015/03/17 14:47:49 terraform-provider-vsphere: 	/usr/local/go/src/net/http/transport.go:661 +0xcbc
2015/03/17 14:47:49 [ERROR] Error walking 'vsphere_virtual_machine.default': 1 error(s) occurred:

* unexpected EOF
2015/03/17 14:47:49 [INFO] Apply walk complete
2015/03/17 14:47:49 [INFO] Writing backup state to: terraform.tfstate.backup
2015/03/17 14:47:49 waiting for all plugin processes to complete...
2015/03/17 14:47:49 [DEBUG] secret_home_path/Downloads/terraform/terraform-provider-vsphere: plugin process exited
```

--- a/vsphere/resource_vsphere_virtual_machine.go
+++ b/vsphere/resource_vsphere_virtual_machine.go
@@ -220,13 +220,8 @@ func resourceVSphereVirtualMachineRead(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
-	d.Set("datacenter", dc)
-	dcFolders, err := dc.Folders(context.TODO())
-	if err != nil {
-		return err
-	}
-
-	vm, err := getVirtualMachine(client, dcFolders.VmFolder, d.Get("name").(string))
+	finder = finder.SetDatacenter(dc)
+	vm, err := finder.VirtualMachine(context.TODO(), d.Get("name").(string))
 	if err != nil {
 		log.Printf("[ERROR] Virtual machine not found: %s", d.Get("name").(string))
 		d.SetId("")
@@ -238,6 +233,7 @@ func resourceVSphereVirtualMachineRead(d *schema.ResourceData, meta interface{})
 	collector := property.DefaultCollector(client.Client)
 	err = collector.RetrieveOne(context.TODO(), vm.Reference(), []string{"summary"}, &mvm)
 
+	d.Set("datacenter", dc)
 	d.Set("memory", mvm.Summary.Config.MemorySizeMB)
 	d.Set("cpu", mvm.Summary.Config.NumCpu)
 
@@ -259,13 +255,8 @@ func resourceVSphereVirtualMachineDelete(d *schema.ResourceData, meta interface{
 		return err
 	}
 
-	d.Set("datacenter", dc)
-	dcFolders, err := dc.Folders(context.TODO())
-	if err != nil {
-		return err
-	}
-
-	vm, err := getVirtualMachine(client, dcFolders.VmFolder, d.Get("name").(string))
+	finder = finder.SetDatacenter(dc)
+	vm, err := finder.VirtualMachine(context.TODO(), d.Get("name").(string))
 	if err != nil {
 		return err
 	}

--- a/vsphere/virtual_machine.go
+++ b/vsphere/virtual_machine.go
@@ -75,7 +75,7 @@ func (vm *VirtualMachine) deployVirtualMachine(c *govmomi.Client) error {
 	}
 
 	vmFolder := dcFolders.VmFolder
-	template, err := getVirtualMachine(c, vmFolder, vm.Template)
+	template, err := finder.VirtualMachine(context.TODO(), vm.Template)
 	if err != nil {
 		return err
 	}
@@ -168,7 +168,7 @@ func (vm *VirtualMachine) deployVirtualMachine(c *govmomi.Client) error {
 		return err
 	}
 
-	newVM, err := getVirtualMachine(c, vmFolder, vm.Name)
+	newVM, err := finder.VirtualMachine(context.TODO(), vm.Name)
 	if err != nil {
 		return err
 	}
@@ -321,17 +321,6 @@ func getDatacenter(f *find.Finder, name string) (*object.Datacenter, error) {
 		}
 		return dc, nil
 	}
-}
-
-// getVirtualMachine finds VirtualMachine or Template object
-func getVirtualMachine(c *govmomi.Client, f *object.Folder, name string) (*object.VirtualMachine, error) {
-	s := object.NewSearchIndex(c.Client)
-	vmRef, err := s.FindChild(context.TODO(), f, name)
-	if err != nil {
-		return nil, err
-	}
-	vm := object.NewVirtualMachine(c.Client, vmRef.Reference())
-	return vm, nil
 }
 
 // getResourcePool finds ResourcePool object

